internal/pkg/service/stream/storage/level/local/writer/csv/csv_test.go

```diff
@@ -308,7 +308,7 @@ func newTestCase(comp fileCompression, syncMode disksync.Mode, syncWait bool, pa
 	}
 
 	return &testcase.WriterTestCase{
-		Name:     fmt.Sprintf("compression=%s,syncMode=%s,wait=%t,parallelWrite=%t", comp.Name, syncMode, syncWait, parallelWrite),
+		Name:     fmt.Sprintf("compression-%s-sync-%s-wait-%t-parallel-%t", comp.Name, syncMode, syncWait, parallelWrite),
 		FileType: model.FileTypeCSV,
```

Comment:
Test name is used as a part of the temp dir name, so I removed Windows sensitive characters.